example/road-traffic/road-traffic-demo/pom.xml

```diff
@@ -72,7 +72,7 @@
                                     ${standalone-path}/v1
                                 </gaffer.rest-api.basePath>
                                 <gaffer.graph.library.path>
```

Comment:
This system property needs updating to use the new gaffer.graph.library.class and gaffer.graph.library.config properties.